packages/pigeon/lib/src/swift/swift_generator.dart

```diff
@@ -9,6 +9,7 @@ import '../ast.dart';
 import '../functional.dart';
 import '../generator.dart';
 import '../generator_tools.dart';
+import '../pigeon_lib.dart';
```

Comment:
Is this necessary?

Reply:
This is for `TaskQueueType`. Per offline discussion, I moved that type to a new file instead.